### DISCUSSION TITLE
Grafana - db query error: pq: permission denied for function dblink

### DISCUSSION THREAD

**Discussion by l0g1k01** [grafana/grafana#41261] [Dashboards, Panels & Visualizations]
I am trying to gather some information from two different prostgresql bases using DBLINK and show them in a panel. It's not working on Grafana.
The script works fine on the manager with any user, Grafana user included. The Grafana user was elevated. And still the connection do not work on Grafana.
It can be solved?

**Comment by Teremus90** [comment]
You have to adjust user's permissions on SQL server side.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
